ui/app/AppLayouts/Chat/panels/communities/HoldingsListPanel.qml

```diff
@@ -0,0 +1,95 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+import AppLayouts.Chat.helpers 1.0
+import AppLayouts.Chat.controls.community 1.0
+
+import SortFilterProxyModel 0.2
+
+import utils 1.0
+
+Control {
+    id: root
+
+    property var model
+    property string introText
+
+    QtObject {
+        id: d
+
+        // By design values:
+        readonly property int defaultHoldingsSpacing: 8
+
+        function holdingsTextFormat(name, amount) {
+            return CommunityPermissionsHelpers.setHoldingsTextFormat(HoldingTypes.Type.Asset, name, amount)
+        }
+    }
+
+    contentItem: ColumnLayout {
+        spacing: root.spacing
+
+        StatusBaseText {
+            Layout.alignment: Qt.AlignHCenter
+            text: root.introText
+            textFormat: Text.StyledText
+        }
+
+        ColumnLayout {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: d.defaultHoldingsSpacing
+
+            Repeater {
+                id: repeater
+                model: root.model
+
+                ColumnLayout {
+                    Layout.alignment: Qt.AlignHCenter
+                    spacing: d.defaultHoldingsSpacing
+
+                    RowLayout {
```

Comment:
This row doesn't take available width into account. With many items/narrow window it will go offscreen.

Reply:
It will not go offscreen since the parent has a scrollview!